app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
This use of class-level variable `people` to store instances of the `Person` class might lead to unexpected behavior. It's better to avoid such global state. Consider refactoring this part of your code. For example, you could use a separate data structure outside of the `Person` class to store `Person` instances.